bfv/bfv_benchmark_test.go

```diff
@@ -74,19 +74,19 @@ func benchEncrypt(testctx *testContext, b *testing.B) {
 	plaintext := NewPlaintext(testctx.params)
 	ciphertext := NewCiphertextRandom(testctx.prng, testctx.params, 1)
 
-	b.Run(testString("Encrypt/Pk/Slow", testctx.params), func(b *testing.B) {
+	b.Run(testString("Encrypt/Pk/Slow/", testctx.params), func(b *testing.B) {
```

Comment:
If it was just "slow", we would not be using it. Since this refers to the "usual" BFV encryption, it could just be `Encrypt/Pk`.

Also, if we follow the notation that we had for parameters until now, it could be `Encrypt/key=pk` (would need to be reflected on the CKKS side). What do you think ?